packages/vesting-base/README.md

```diff
@@ -1,16 +1,32 @@
-# Astroport Generator Vesting
+# Astroport Vesting
 
-The Generator Vesting contract progressively unlocks ASTRO that can then be distributed to LP stakers via the Generator contract.
+The Vesting contract progressively unlocks VESTED that can then be distributed to LP stakers.
```

Comment:
`...progressively unlocks VESTED that can...` -> `...progressively unlocks vested NTRN that can...`? VESTED looks like an exact token symbol, but I don't think it will look like this, will it?